packages/inter-protocol/test/psm/test-governedPsm.js

```diff
@@ -116,3 +116,67 @@ test('psm block offers w/charter', async t => {
     { message: 'not accepting offer with description "wantMinted"' },
   );
 });
+
+test('psm block offers w/charter via invitationMakers', async t => {
+  const electorateTerms = { committeeName: 'EnBancPanel', committeeSize: 3 };
+  const timer = buildManualTimer(t.log, 0n, { eventLoopIteration });
+
+  const { knutIssuerKit, zoe, psm, committeeCreator, psmCharterCreatorFacet } =
+    await setupPsm(t, electorateTerms, timer);
+
+  const knutBrand = knutIssuerKit.brand;
+
+  // onchain, the invitations get sent to the committee member via their
+  // registered deposit facets. We'll skip that here.
+  const invitations = await E(committeeCreator).getVoterInvitations();
+
+  const cmInvitation = await E(
+    psmCharterCreatorFacet,
+  ).makeCharterMemberInvitation();
+
+  const cmSeat = E(zoe).offer(cmInvitation);
+  const invitationMakers = await E(cmSeat).getOfferResult();
```

Comment:
Is that right? above, the offer handler seems to return `{ invitationMakers }`, not `invitationMakers`.

Reply:
yeah, that's got to be `{ invitationMakers }`. I wish we had time to improve Zoe types because this should be inferred.

Reply:
fixed